### PR TITLE
Add some bool attributes

### PR DESCRIPTION
**Pull request by nrjais**
Contribution to #358 
Added
```
* __int__
* __ceil__
* __floor__
* __round__
* __float__
```

### CODE REVIEW

**Comment by nrjais** [comment]
@veera83372 done

File: vm/src/obj/objbool.rs [hunk 1]
```diff
@@ -60,6 +65,26 @@ pub fn get_value(obj: &PyObjectRef) -> bool {
     }
 }
 
+fn bool_to_int(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
+    arg_check!(vm, args, required = [(obj, Some(vm.ctx.int_type()))]);
+
+    if let PyObjectPayload::Integer { value } = &obj.borrow().payload {
```
Comment (windelbouwman) [inline vm/src/obj/objbool.rs]: You can use objint::get_value to perform this logic for you. Please use it, to make the code smaller.

**Comment by windelbouwman** [comment]
Since bool inherits from int, are methods like ceil required here?

File: vm/src/obj/objbool.rs (hunk 1 above)
Comment (nrjais) [reply vm/src/obj/objbool.rs]: Done

**Comment by cthulahoops** [comment]
Closing: these are inherited from int.

**Pull request closed without merging**